From 6ff88695e8cd843c0872cbd737048e0957226521 Mon Sep 17 00:00:00 2001
From: Patkar Rutuparna <rutuparnax.patkar@intel.com>
Date: Tue, 20 Apr 2021 19:19:33 -0400
Subject: [PATCH 6/7] mcdma: enable meson build for mcdma pmd and example  apps

This patch has the changes for enabling meson build for mcdma
driver and example applications.

Signed-off-by: Venkata Suresh Kumar P <venkata.suresh.kumar.p@intel.com>
---
 drivers/net/meson.build | 1 +
 examples/meson.build    | 2 ++
 2 files changed, 3 insertions(+)

diff --git a/drivers/net/meson.build b/drivers/net/meson.build
index 266448f..f737101 100644
--- a/drivers/net/meson.build
+++ b/drivers/net/meson.build
@@ -26,6 +26,7 @@ drivers = ['af_packet',
 	'ixgbe',
 	'kni',
 	'liquidio',
+	'mcdma',
 	'memif',
 	'mlx4',
 	'mlx5',
diff --git a/examples/meson.build b/examples/meson.build
index 1f2b6f5..9214f94 100644
--- a/examples/meson.build
+++ b/examples/meson.build
@@ -25,6 +25,8 @@ all_examples = [
 	'l2fwd-keepalive', 'l3fwd',
 	'l3fwd-acl', 'l3fwd-power',
 	'link_status_interrupt',
+	'mcdma-test/sample',
+	'mcdma-test/perfq',
 	'multi_process/client_server_mp/mp_client',
 	'multi_process/client_server_mp/mp_server',
 	'multi_process/hotplug_mp',
-- 
1.8.3.1

